settings/default/main.lua

```diff
@@ -207,6 +207,35 @@ xi.settings.main =
     EXPLORER_MOOGLE_LV              = 10, -- Enables Explorer Moogle teleports and sets required level. Zero to disable.
     HALLOWEEN_2005                  = 0,  -- Set to 1 to Enable the 2005 version of Harvest Festival, will start on Oct. 20 and end Nov. 1.
     HALLOWEEN_YEAR_ROUND            = 0,  -- Set to 1 to have Harvest Festival initialize outside of normal times.
+    EGG_HUNT                        =
+    {
+        START                       = { DAY = 6,  MONTH = 4 },
+        FINISH                      = { DAY = 17, MONTH = 4 },
+
+        -- Default era is 2005
+        ERA_2006 = false, -- Orphic Egg
+        ERA_2007 = false, -- Jeweled Egg and Egg Helm
+        ERA_2008 = false, -- Tier 2 nation eggs, allows trading Hard-Boiled Eggs
+        ERA_2009 = false, -- Egg Buffet set
+        -- 2009, 2010, 2011 and 2012 are identical
+        ERA_2013 = false, -- Prinseggstarta
+        ERA_2014 = false, -- Hatchling Shield, Copse Candy, Cracker
+        ERA_2015 = false, -- Rabbit Cap, show Rabbit Cap wearing NPCs
+        ERA_2018 = false, -- Allows trading Sairui-Ran x99 and Imperial Egg
+        ERA_2019 = false, -- Allows trading Apkallu Egg
+
+        -- Consolation prizes for repeating combinations where
+        -- the player has already received the relevant reward
+        MINOR_REWARDS = true,
+
+        -- Set custom combinations, eg. WORD = 12345
+        -- Where WORD  is an arrangement of lettered eggs
+        -- Where 12345 is the itemID for the reward
+        BONUS_WORDS =
```

Comment:
This and the Era settings above.. You went above and beyond 👏

Reply:
I think this creates enough event settings to warrant being separated from main.lua though.

The most recent event to be added was starlight celebration, which did its its is primary event file.

https://github.com/LandSandBoat/server/blob/base/scripts/globals/events/starlight_celebration.lua#L14

I forget if we already discussed doing it, but I think we've enough here to warrant a new `settings/default/seasonal.lua` file to move everything seasonal event related to instead of storing in `main.lua`

I think we can merge as-is then do that migration after.